api/views.py

```
@@ -0,0 +1,115 @@
+from patients_core.models import Patient
+from diab_encounters.models import Encounter
+from diab_labs.models import LabResult
+from diab_medications.models import MedicationOrder
+from clinical_refs.models import ClinicalReference
+from ai_summarizer.models import AISummary
+from .serializers import (
+    PatientSerializer, EncounterSerializer, LabResultSerializer,
+    MedicationOrderSerializer, ClinicalReferenceSerializer, AISummarySerializer
+)
+
+SYSTEM_USER_ID = UUID(getattr(settings, 'SYSTEM_USER_ID'))
```

Ruff (0.12.2)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)